flink-contrib/flink-storm/src/main/java/org/apache/flink/storm/wrappers/WrapperSetupHelper.java

```diff
@@ -224,7 +224,7 @@ static synchronized TopologyContext createTopologyContext(
 	 *            OUTPUT: A map from all component IDs to there output streams and output fields.
 	 * 
 	 * @return A unique task ID if the currently processed Spout or Bolt ({@code componentId}) is equal to the current
-	 *         Flink operator ({@link operatorName}) -- {@code null} otherwise.
+	 *         Flink operator ({@param operatorName}) -- {@code null} otherwise.
```

Comment:
Ok, let's use code.

http://stackoverflow.com/questions/1667212/reference-a-method-parameter-in-javadoc